umpost/um2netcdf.py

```diff
@@ -406,6 +360,64 @@ def cubewrite(cube, sman, compression, use64bit, verbose):
         sman.write(cube, zlib=True, complevel=compression, fill_value=fill_value)
 
 
+def fix_forecast_reference_time(cube):
+    # If reference date is before 1600 use proleptic gregorian
+    # calendar and change units from hours to days
+    try:
+        reftime = cube.coord('forecast_reference_time')
+        time = cube.coord('time')
+        refdate = reftime.units.num2date(reftime.points[0])
+        assert time.units.origin == 'hours since 1970-01-01 00:00:00'
+
+        if time.units.calendar == 'proleptic_gregorian' and refdate.year < 1600:
+            convert_proleptic(time)
+        else:
+            if time.units.calendar == 'gregorian':
+                new_calendar = 'proleptic_gregorian'
+            else:
```

Comment:
Just noticed this isn't using the added constants (e.g. `PROLEPTIC_GREGORIAN`, or otherwise the `cf_units` alternatives)

Reply:
This is updated in 16a3768a.